apps/prairielearn/src/lib/client/page-context.ts

```diff
@@ -1,12 +1,8 @@
 import { z } from 'zod';
 
 import {
-  type StaffCourse,
-  type StaffCourseInstance,
```

Comment:
👀 this deletion caught my eye. AFAICT, we can't refer to the type for "`StudentCourseInstance` with non-null `short_name`" directly by name. I see in the other PR you're using `StaffCourseInstanceContext['course']`. Is that how you're envisioning referring to this type? IMO that's fine, naming things explicitly is hard 🙂

Reply:
Yes, that's how I'm thinking about it.